src/manifest/mod.ts

```diff
@@ -111,21 +109,6 @@ export class SlackManifest {
       );
     }
 
-    if (def.features?.appHome) {
```

Comment:
We should probably not be setting these overrides all in one place since it's not clear that there's a distinction for ROSI vs. Remote apps. Prefer to handling anything ROSI specific in the `assignRunOnSlackManifestProperties `and Remote specific in `assignRemoteSlackManifestProperties`